content/source/docs/enterprise/api/oauth-clients.html.md

```diff
@@ -111,6 +111,100 @@ curl \
 }
 ```
 
+## Update an OAuth Client
+
+`PATCH /oauth-clients/:id`
+
+Parameter            | Description
+---------------------|------------
+`:id`                | The ID of the OAuth Client to update.
+
```

Comment:
```suggestion

When a VCS service provider changes their API URL, use this endpoint to update the value of `data.attributes.api-url`. Use caution when changing other attributes with this endpoint; editing an OAuth client that workspaces are currently using can have unexpected effects.

```

...or something like that? I feel like I remember from our Slack conversation that there were some possible hazards when pulling the tablecloth out from under the dishes like that.